concordium-consensus/src/Concordium/ProtocolUpdate/P7.hs

```diff
@@ -53,9 +57,11 @@ updateRegenesis ::
     BlockPointer (MPV m) ->
     m (PVInit m)
 updateRegenesis Reboot = Reboot.updateRegenesis
```

Comment:
this should reboot P8, not P7. I think the constraint in line 49 is probably also wrong.